### DISCUSSION TITLE
一人企业、Super individual 在 706 形成支持社群的实践

### DISCUSSION THREAD

**Discussion by Demian101** [creatorsand/.github#52] [Ideas]
什么是 一人企业？

> - Super individual -- 不是各方面的通才，而是全面的 T 型人才，各方涉猎，一方专精 （706 colearning）
> - 一人企业 community - 法务、财务、技术、设计、推广 ... 各安其位，各个领域的专才彼此协作于 fancy 的 ideas ，目标是成立一个个小而美的公司或实现一个个有趣值得推进的 ideas 。

**Company for one VS super individual**
- super individual 某种层面还是在用必要劳动时间换钱，这固然没问题，但是一人企业更强调产品的复用性，强调 passive income

----

706 `company for one` community 能提供什么？

1. 有些东西是 T 型人才需要学习的，比如设计、AI、Coding....   ——Creators co-learning Group 提供
2. 有些东西是社群提供的，比如法务、财税、股权... 这些只要知道以什么价格、找什么人、什么时候开始外包 (outsource) 就可以了  —— 706 Creators community 提供。

----

company for one 的社群模式还是很有潜力的: 

- 在 706 社群中，信息流动的**低效率**和**优质信息交换的困难**仍是最大的痛点。 
	- 好的 ideas 找不到靠谱的合适的人；
	- 人在社群里不知道能做什么；
- 有了 `company for one` 的社群支持，可以迅速聚集起一些创业者，以 company for one 的模式去 学习成长 对接和创造。
- 解决的痛点：
	- 洞察市场动向和需求的人找不到 Devs 来 Build；
	- Builders 产出了一堆不美，也没有市场需求的炫技垃圾；
	- 设计师天马行空，但找不到其心仪的产品来付诸美学实践。
	- 法务、财税、市场、运营从业者受困于平台，只能做一个螺丝钉....
 - 我们期望的流程：
	 - 1. (任何人)嗅到 ideas，分享-> 有效流转 -> 社群打磨
	 - 2. PM、Business 分析市场，可行性验证，进一步打磨 ideas 
	 - 3. 独立开发者+设计师+ AI 加持迅速接入完成 MVP
	 - BD 寻找对接客户
	 - 后续成立公司，法务财税进场支持
	 - ... 

----

<br />
<br />
<br />
<br />
<br />
<br />
<br />

### 具体关于 Company for one 的方法论介绍：

1. 针对细分市场，目标群体，受众群体，而不是取悦所有人。理解他们的需求，解决他们的问题或心理问题。 
2. 一人企业更要展示独特性：技能和专长是可以复制的，但要复制一个人的个性和风格基本是不可能的。 
3. 提供情感和情绪：所有一人企业都应提供的客户服务主要侧重情感和舒适感。吸引客户的关键在于摆脱乏味。开发兴趣，激发受众对你的企业和你的品牌个性的强烈感情。 
4. 注意力：对一个产品或服务的喜爱会建立情感联系，这会维持注意力。 
5. 成为专家，建立权威：共享知识，让他们认为你是某个领域的专家，成为长期客户。 

比起思考“我能做什么产品”或者“我能提供什么服务”，我们更应该首先思考“我想要什么样的生活”以及“我想怎样过每一天”

运营方面： 
1. 一人企业需要尽快发布第一个产品，因为要尽早得到市场反馈，以便修改，尽快盈利
2. 最注重的应当是最低可行利润 MVPr，不是毛收入

只要你能给别人提供某种价值，你就已经能成立一家麻雀虽小五脏俱全的“一人企业”——精炼，聆听客户，迅速迭代，永远可以在市场中找到自己的位置，不会被大公司击败

本书的理念是，你还需要设定一个上限。我的理解是，工作是做不完的、钱是赚不完的、百尺竿头总可以更进一步，必须严肃地思考一下，你到底什么时候停下来
- 这一点上，“一人企业”不走增长黑客的路线，因为增长黑客的本质是增长。对增长黑客企业来说，增长是衡量成功与否的唯一标准。但“一人企业”是反增长的，不在乎虚荣指标，而更在乎过程，做事开心
- 在企业/产品的发展阶段里面，如果要做大，都会走向“平台化” ——要想清楚自己想不想做大，做大不是必然的路径，相比做大带来的更多权责利，应该允许也需要小而美
- 上限能帮你知道多少是足够的，而后即可获得真正的自由，有自由对别人期待的事或者对你毫无用处的机会说不。

**Comment by Demian101** [comment]
<img width="536" alt="image" src="https://github.com/CreatorsDAO/.github/assets/33189338/937b9a78-ccfa-4ace-a6cb-90042c376686">

**Comment by kabudafax** [comment]
Hello, I think this is a great idea. Is there any progress? I'm a developer and I'd like to meet more people to discuss the potential  product.

**Comment by Cobb9527** [comment]
蹲一个后续进展~
